index.html

```diff
@@ -216,5 +223,33 @@ <h2 id="about-text">About me</h2>
       </div>
     </div>
     </section>
+    <!-- About section end here  -->
+
+    <!-- footer section starts from here -->
+    <footer class="form">
+        <div class="form-description">
+            <h2 id="form-desc">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+        <div class="form">
+            <form action="https://formspree.io/f/mlevbzyw" method="post">
+                <input type="text" id="fname" name="fname" maxlength="30" placeholder="    Full name"><br>
+                <input type="email" id="email" name="email" placeholder="   Email adress">
+                <textarea name="message" maxlength="500"id="textarea"> 
+ Enter text here</textarea>
```

Comment:
- [ ] As the project requires, the user should not be able to submit the form if any of the inputs are empty. Kindly make sure that all the inputs are required in order to submit the form.
 - [ ] The `textarea` input should have a placeholder like the other inputs. By adding a text inside the tag, when the page loads, there is already a sample text and the user can accidentally submit the form without writing a text. Kindly remove this text and use it as a placeholder.

Reply:
Made form input filed required and added text in placeholder for textrarea of form. Please check.